rules/windows/credential_access_remote_sam_secretsdump.toml

```diff
@@ -13,7 +13,7 @@ Identifies remote access to the registry to potentially dump credential data fro
 registry hive in preparation for credential access and privileges elevation.
 """
 from = "now-9m"
-index = ["winlogbeat-*", "logs-system.*", "logs-endpoint.events.*"]
+index = ["logs-endpoint.events.*"]
```

Comment:
Sequence dropped as part of https://github.com/elastic/detection-rules/pull/2953, removing unnecessary indexes